ui/pages/bridge/prepare/prepare-bridge-page.tsx

```diff
@@ -214,8 +217,27 @@ const PrepareBridgePage = () => {
   }, [rotateSwitchTokens]);
 
   useEffect(() => {
-    // Reset controller and inputs on load
-    dispatch(resetBridgeState());
+    if (activeQuote) {
+      // Get input data from active quote
+      const { srcAsset, destAsset, destChainId } = activeQuote.quote;
+      const quoteSrcToken = fromTokens[srcAsset.address.toLowerCase()];
+      const quoteDestChainId = `0x${decimalToHex(destChainId)}`;
```

Comment:
You can use `decimalToPrefixedHex` from `shared/modules/conversion.utils.ts`

Reply:
Didn't know this existed, thanks! Done.